web/modules/custom/dpl_patron_reg/src/Plugin/Block/PatronRegistrationBlock.php

```diff
@@ -104,7 +104,7 @@ public function build(): array {
 
     // Get user info endpoint from OpenIdConnect configuration.
     $configuration = $this->configFactory->get('openid_connect.settings.adgangsplatformen');
-    $userInfoEndpoint = $configuration->get('settings')['userinfo_endpoint'];
+    $userInfoEndpoint = $configuration->get('settings')['userinfo_endpoint'] ?? '/';
```

Comment:
Consider if there is a mixing between handling of default urls.

This seems to uses a `/` as a default value for a url. Other places a blank space is used.